LAST_RELEASE

```diff
@@ -0,0 +1 @@
+1.1.5
```

Comment:
Is this now a hand-maintained file?
How do we minimize the risk of this being stale?

Reply:
Added to release instructions. Eventually, once we automate creating a release on GitHub, it can be part of that script.